views/login/questions/dateOfResidence.pug

```diff
@@ -0,0 +1,48 @@
+extends ../../base
+
+block variables
+  -var title = __('Enter date you became a resident of Canada')
+  -var dobDay = hasData(body, 'dobDay') ? body.dobDay : ''
+  -var dobMonth = hasData(body, 'dobMonth') ? body.dobMonth : ''
+  -var dobYear = hasData(body, 'dobYear') ? body.dobYear : ''
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('For income tax, you become a resident by building strong ties to Canada as your home. Usually this happens on the date you arrive in Canada.')}
+
+    p #{__('If you ever lost permanent resident status, enter the date you returned to Canada.')}
+  div
+    form.cra-form(method='post')
+
+      div.dob-container(class={['has-error']: errors && errors.dobDay})
+        fieldset.fieldset(role="group", aria-describedby="dobHint")
+          legend
+            h2 #{__('Date you became a resident of Canada')}
+
+          span#dobHint.cra-form-message #{__('For Example:')} 21 05 2006
+
+          if errors
+            ul
+              each err in errors
+                - if (err.param.startsWith("dob"))
+                  li
+                    +validationMessage(err.msg, err.param)
+
+          div.w-1-4.d--ib
+            label(for='dobDay') #{__('Day')}
+            input#dobDay(name='dobDay', type='text', value=dobDay, autocomplete='bday-day', aria-describedby=(errors && errors.dobDay ? 'dobDay-error' : false))
+
+          div.w-1-4.d--ib
+            label(for='dobMonth') #{__('Month')}
+            input#dobMonth(name='dobMonth', type='text', value=dobMonth, autocomplete='bday-month', aria-describedby=(errors && errors.dobMonth ? 'dobMonth-error' : false))
+
+          div.w-1-2.d--ib
+            label(for='dobYear') #{__('Year')}
+            input#dobYear(name='dobYear', type='text', value=dobYear, autocomplete='bday-year', aria-describedby=(errors && errors.dobYear ? 'dobYear-error' : false))
```

Comment:
Ooh, actually we should remove the b-day autocomplete stuff here. (If we're not asking for a date of birth, the browser might do the wrong thing.)

Reply:
Oh, and if you could remove the `autocomplete` fields on the other "dates" in the security questions we have, that would be appreciated.